CMakeLists.txt

```diff
@@ -441,13 +478,13 @@ set(DEPENDENCY_EIGEN_INCLUDE ${CMAKE_CURRENT_LIST_DIR}/third_party_libs/eigen CA
 set(DEPENDENCY_TINYGLTF_INCLUDE ${CMAKE_CURRENT_LIST_DIR}/third_party_libs/tinygltf CACHE PATH "Path to include directory.")
 
 if(UNIX)
-	set(DEPENDENCY_BOOST_CHRONO_LIBRARY ${DEPENDENCY_BOOST_LIBRARY_LOCATION}/libboost_chrono.a CACHE PATH "Path to library." FORCE)
```

Comment:
@SlawekNowy Why are the shared versions of boost needed?

Reply:
I do not want to duplicate dependencies here. And something else also uses boost.
I don't remember exactly what.